dockstore-integration-testing/src/test/java/io/dockstore/client/cli/ExtendedTRSOpenApiIT.java

```diff
@@ -72,27 +98,227 @@
  * Extra confidential integration tests, focuses on proposed GA4GH extensions
  * {@link BaseIT}
  */
-@ExtendWith(SystemStubsExtension.class)
-@ExtendWith(MuteForSuccessfulTests.class)
-@ExtendWith(BaseIT.TestStatus.class)
+@LocalstackDockerProperties(imageTag = IMAGE_TAG, services = { ServiceName.S3 }, environmentVariableProvider = LocalStackTestUtilities.LocalStackEnvironmentVariables.class)
+@ExtendWith({ SystemStubsExtension.class, MuteForSuccessfulTests.class, BaseIT.TestStatus.class, LocalstackDockerExtension.class })
 @Tag(ConfidentialTest.NAME)
+@Tag(LocalStackTest.NAME)
```

Comment:
May be worth renaming the class. At a glance, it looks like this class only tests metrics related endpoints so maybe 
`ExtendedMetricsTRSApiIT` for example. 
(i.e. nudge tests of other extended TRS api stuff to go into a different class)